PeripheryDemoTests/PeripheryDemoTests.swift

```diff
@@ -1,14 +1,14 @@
 //
-//  Periphery_DemoTests.swift
-//  Periphery DemoTests
+//  PeripheryDemoTests.swift
+//  PeripheryDemoTests
 //
 //  Created by Sheikh Atif on 14/07/24.
 //
 
 import XCTest
-@testable import Periphery_Demo
+@testable import PeripheryDemo
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/SheikhAtifIBM/Periphery-Demo/blob/8ae70dbe002b70726bb9e9b9dbf964a049fbb194/PeripheryDemoTests/PeripheryDemoTests.swift#L9"></span>Imported module 'PeripheryDemo' is unused</td>
    </tr>
  </tbody>
</table>